core/tests/protractor/libraryPagesTour.js

```diff
@@ -58,12 +58,6 @@ describe('Library pages tour', function() {
     users.logout();
   });
 
-  it('visits the recent explorations page', function() {
-    visitLibraryPage();
-    element(by.css('.protractor-test-library-recently-published')).click();
-    expect(browser.getCurrentUrl()).toContain('library/recently_published');
```

Comment:
I think that this page should still exist (e.g. for moderators to look at), we should just remove the category from the default library page. Is that possible?